File: app/src/main/java/com/pluu/webtoon/event/ThemeEvent.java
package com.pluu.webtoon.event;

import android.support.annotation.ColorInt;

/**
 * Theme Change Event
 * Created by PLUUSYSTEM-NEW on 2015-11-22.
 */
public class ThemeEvent {

	private final int color, darlColor;

	public ThemeEvent(@ColorInt int color, @ColorInt int darlColor) {
		this.color = color;
		this.darlColor = darlColor;
	}

	public int getColor() {
		return color;
	}

	public int getDarlColor() {
		return darlColor;
	}
}

File: app/src/main/java/com/pluu/webtoon/event/ThemeEvent.kt
package com.pluu.webtoon.event

import androidx.annotation.ColorInt
import java.io.Serializable

/**
 * Theme Change Event
 * Created by pluu on 2017-04-18.
 */
class ThemeEvent(
    @ColorInt val color: Int,
    @ColorInt val variantColor: Int
) : Serializable
